File: Sillage.API/DTOs/FragranceEnrichmentResponse.cs
using Sillage.API.Models;

namespace Sillage.API.DTOs;

public record FragranceEnrichmentResponse(
    string Name,
    string Brand,
    string Gender,
    string OilType,
    List<string> GeneralNotes,
    string Description,
    List<string> MainAccords
);

File: sillage-web/src/types/FragranceEnrichmentResponse.ts
export type FragranceEnrichmentResponse = {
    name: string
    brand: string
    gender: string
    oilType: string
    description: string
    generalNotes: string[]
    mainAccords: string[]
    seasonRanking: { name: string; score: number }[]
    occasionRanking: { name: string; score: number }[]
}